### PR TITLE
feature/socials

### PR DESCRIPTION
**Pull request by KianMiridoozini**
Users can now update their profiles and add links to their social media.

### CODE REVIEW

File: app/config/dependencies.php [hunk 1]
```diff
@@ -10,6 +10,7 @@
 require_once __DIR__ . '/../repositories/CommentRepository.php';
 require_once __DIR__ . '/../repositories/UserRepository.php';
 require_once __DIR__ . '/../repositories/BadgeRepository.php';
+require_once __DIR__ . '/../repositories/SocialsRepository.php';
```
Comment (jayb2302) [inline app/config/dependencies.php]: Can we name it SocialLinkRepsitory ?
beacause the model is SocialLink,
same goes for SocialLinkService & sociallinks.js or social-links.js
🫣

Comment (KianMiridoozini) [reply app/config/dependencies.php]: yes, However the reason for this naming was the in order to follow the rest of the naming conventions for repositories.

1 word + Repository

Comment (jayb2302) [reply app/config/dependencies.php]: The rest of it is exactly as the model name and then Repository, service and controller

Comment (KianMiridoozini) [reply app/config/dependencies.php]: fixed

**Review by jayb2302 — APPROVED** [review]
You've been approved ✅

**Pull request merged**